src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -106,6 +107,34 @@ def visit(self, node: concepts.RootNode, **kwargs: Any) -> Any:
         ...
 
 
+def _collect_symbols_in_domain_expressions(
+    ir: gtir.Node, ir_params: Sequence[gtir.Sym]
+) -> set[str]:
+    """
+    Collect symbols accessed in domain expressions that also appear in the paremeter list.
```

Comment:
I would also add for what is it used.
Something like:
This function is used to identify all program arguments that are used in domain expressions.
They have to be passed as symbols (instead of scalars) such that they can be used as bounds.

Reply:
Done.